packages/app/src/systems/Tweet/hooks/useTwitterDialog.ts

```diff
@@ -0,0 +1,23 @@
+import { atom, useAtom } from 'jotai';
+
+const TWITTER_DIALOG_KEY = 'swayswap-dialog';
```

Comment:
why do we need to persist at localstorage here?

Reply:
The popup should be open only once, after the user takes an action, share or dismiss we never show again.